Sources/StreamChatUI/Resources/en.lproj/Localizable.strings

```diff
@@ -22,6 +22,9 @@
 "message.actions.user-mute" = "Mute User";
 "message.actions.resend" = "Resend";
 
+"messageList.typingIndicator.typing-singular" = "is typing";
+"messageList.typingIndicator.typing-plural" = "more are typing";
```

Comment:
Using singular for one and plural for 2 and more is sufficient in English, but not for other languages. A proper pluralisation would be nice here. I'm not sure how L10n works, but the standards iOS way is to use stringsdict file.